src/odemis/acq/move.py

```diff
@@ -875,40 +919,84 @@ def _doCryoSwitchSamplePosition(future, target):
                 raise CancelledError()
             future._task_state = FINISHED
 
-# Note: this transformation consists of translation of along x and y
-# axes, and 7 degrees rotation around rx, and 180 degree rotation around rz.
-# The rotation angles are constant existing in "FM_POS_ACTIVE" metadata,
-# but the translation are calculated based on the current position and some
-# correction/shifting parameters existing in metadata "FM_POS_ACTIVE". 
-# This correction parameters can change every session. They are calibrated
-# at the beginning of each run.
+
+def check_keys_in_md_calib(calibrated_values: dict) -> bool:
+    """
+    Check the required keys in the calibrated values.
+    :param calibrated_values:Is a dictionary containing the calibrated values to carry out the transformation
+        from SEM to FM.
+    :return: True if all keys are in the calibrated values, False otherwise.
```

Comment:
I have change the doc string as we need to raise instead of returning False. This function is modified to include no return value. The function just raises if the keys are not proper.